web_server.py

```
@@ -45,409 +71,507 @@ def _mask_id(value: str) -> str:
+    # search results that then fail to open).
     try:
-        profiles_del, events_del = db.clear_user_memory(user_id)
-        return JSONResponse({
-            "ok": True,
-            "profiles_deleted": profiles_del,
-            "events_deleted": events_del,
-        })
+        # 1. Drop index rows for this entity so search/recall stop surfacing them.
+        # `MemoryIndex.delete` 现在按复合主键 (entity_type, entity_id, folder,
+        # base_dir, id) 删单行——只传裸 id 会一行都删不掉。`row` 自身就携带
+        # 完整 entity 维度，照搬即可。
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)